server/src/analyzer/program.ts

```diff
@@ -1106,13 +1106,17 @@ export class Program {
             }
 
             for (const sourceFileInfo of this._sourceFileList) {
-                this._bindFile(sourceFileInfo);
+                // "Find symbols" will only include references from user code
+                // unless the file is explicitly opened in the editor.
+                if (this._isUserCode(sourceFileInfo) || sourceFileInfo.isOpenByClient) {
```

Comment:
I think the `isOpenByClient` could be dropped, but otherwise this seems correct to me.